web/static/salesforce/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>AutoKitteh - Salesforce</title>
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/static/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/static/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/static/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/static/site.webmanifest" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <meta name="robots" content="noindex, nofollow" />
+    <script src="form.js" defer></script>
+  </head>
+
+  <body>
+    <a href="/" target="_self">
+      <img class="banner" src="/static/banner.svg" alt="AutoKitteh" />
+    </a>
+
+    <table class="title">
+      <td>
+        <img class="logo" src="/static/images/salesforce.svg" alt="Logo" />
+      </td>
+      <td><h1>Salesforce - Initialize Connection</h1></td>
+    </table>
+
+    <form method="post" action="/salesforce/save">
+      <h2>Query Params From the URL</h2>
+
+      <div class="form-group">
+        <label for="cid">Connection ID</label>
+        <input type="text" name="cid" id="cid" readonly />
+      </div>
+
+      <div class="form-group">
+        <label for="origin">Request Origin</label>
+        <input type="text" name="origin" id="origin" readonly />
+      </div>
+
+      <h2>Authentication Type</h2>
+
+      <div class="form-group">
+        <select name="auth_type" id="authType">
+          <option value="oauthPrivate">Private OAuth 2.0</option>
+        </select>
```

Comment:
If there's no user choice, we might as well make it a pre-populated read-only field (and hidden in the official web UI, but that's beyond the scope of this PR).

```suggestion
        <input type="text" name="auth_type" value="oauthPrivate" readonly />
```